projects/packages/search/src/dashboard/components/card/compact.jsx

```diff
@@ -1,14 +1,13 @@
 /*eslint lodash/import-scope: [2, "method"]*/
 import clsx from 'clsx';
-import assign from 'lodash/assign';
 import { Component } from 'react';
 import Card from 'components/card';
 
 export default class CompactCard extends Component {
 	static displayName = 'CompactCard';
 
 	render() {
-		const props = assign( {}, this.props, {
+		const props = Object.assign( {}, this.props, {
```

Comment:
Why not this?
```suggestion
		const props = {
			...this.props,
```

Reply:
Here? No reason I know of. `Object.assign` is a bit more of a direct mapping from the previous code.

In some of `projects/plugins/jetpack/modules/`, we'd want to check whether the target browsers all support spread syntax.